manifest.json

```diff
@@ -1,36 +1,53 @@
 {
-    "manifest_version": 2,
-    "name": "Whimsy",
-    "version": "2.0.0",
-    "description": "Add some Whimsy to your Firefox! 🦄🌈",
-    "homepage_url": "https://github.com/bwinton/whimsy/tree/webextension",
-    "author": "Blake Winton <bwinton@latte.ca>",
-    "icons": {
-        "48": "icon.png",
-        "64": "icon64.png"
-    },
+   "manifest_version": 2,
+   "name": "Whimsy",
+   "version": "2.0.0",
+   "description": "Add some Whimsy to your Firefox! 🦄🌈",
+   "homepage_url": "https://github.com/bwinton/whimsy/tree/webextension",
+   "author": "Blake Winton <bwinton@latte.ca>",
+   "contributors": [
+      "Daniela Arcese",
+      "Erica Wright",
+      "Bryan Clark",
```

Comment:
Given that this is a re-write, I think we could take everyone but you and Erica out of the contributors…  🙂